modules/ting_search/includes/ting_search.admin.inc

```diff
@@ -39,19 +39,59 @@ function ting_search_admin_settings($form_state) {
     '#default_value' => variable_get('ting_search_number_of_facets', 25),
   );
 
-  $form['ting_search']['ting_search_register_serie_title'] = array(
+  $form['ting_search']['search_strings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Search strings'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['ting_search']['search_strings']['ting_search_register_serie_title'] = array(
     '#type' => 'textfield',
     '#title' => t('Serie title'),
-    '#description' => t('Specify the searchstring to be used for searching against serie titles. Use @serietitle as a placeholder for the serietitle'),
-    '#default_value' => variable_get('ting_search_register_serie_title', 'phrase.titleSeries="@serietitle"'),
+    '#description' => t(
+      'Specify the search-string to be used for searching against serie titles. 
+      Use @serietitle as a placeholder for the serietitle'
+    ),
+    '#default_value' => variable_get(
```

Comment:
Why the multi line variable_get in this file?

Reply:
To avoid long lines of code.